src/sentry/replays/endpoints/organization_replay_index.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases.organization import NoProjects, OrganizationEndpoint
+from sentry.models.organization import Organization
+from sentry.replays.utils import proxy_replays_service
+
+
+class OrganizationReplayIndexEndpoint(OrganizationEndpoint):
+    private = True
+
+    def get(self, request: Request, organization: Organization) -> Response:
+        if not features.has("organizations:session-replay", organization, actor=request.user):
+            return Response(status=404)
+
+        try:
+            filter_params = self.get_filter_params(request, organization, date_filter_optional=True)
+        except NoProjects:
+            return Response([])
+
+        for key, value in request.query_params.items():
+            if key not in filter_params:
+                filter_params[key] = value
+
+        return proxy_replays_service("GET", "/api/v1/replays/", filter_params)
```

Comment:
You mentioned that you're proxying to cloud-run functions right now. Longer term will this service be something we add to`devservices` and self-hosted as an optional backend service?